### PR TITLE
Add bookmarks to session details screen

### PR DESCRIPTION
**Pull request by brunohensel**
This PR adds bookmark capability on the sessions details screen for Android. Solving part of  #405 

https://user-images.githubusercontent.com/45492687/231000299-fc7c7508-286e-4961-b5a5-31b66bd0de1d.mp4

### CODE REVIEW

File: androidApp/src/main/java/dev/johnoreilly/confetti/ui/Bookmark.kt [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+import androidx.compose.material3.Icon
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.unit.dp
+
+@Composable
+fun Bookmark(
+    isBookmarked: Boolean,
+    onBookmarkClick: () -> Unit,
```
Comment (marcellogalhardo) [inline androidApp/src/main/java/dev/johnoreilly/confetti/ui/Bookmark.kt:17]: Nit: `onBookmarkChange: (Boolean) -> Unit` where `Boolean` is the flip value, similar to how a Toggle button does in Compose.

File: androidApp/src/main/java/dev/johnoreilly/confetti/ui/Bookmark.kt [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+import androidx.compose.material.icons.outlined.BookmarkAdd
+import androidx.compose.material3.Icon
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.unit.dp
+
+@Composable
+fun Bookmark(
+    isBookmarked: Boolean,
```
Comment (marcellogalhardo) [inline androidApp/src/main/java/dev/johnoreilly/confetti/ui/Bookmark.kt:16]: Nit: Missing default `Modifier` parameter.

**Comment by joreilly** [comment]
@brunohensel  thanks for the contribution!  Just fyi in case you hadn't seen it but you'll need to resolve some merge conflicts and push again.

**Comment by brunohensel** [comment]
> @brunohensel thanks for the contribution! Just fyi in case you hadn't seen it but you'll need to resolve some merge conflicts and push again.

Done!

**Pull request merged**